setup.cfg

```diff
@@ -38,7 +38,7 @@ toml =
 
 [entry_points]
 console_scripts =
-    bandit = bandit.cli.main:main
```

Comment:
This is not how you add a new extra. L33 above starts the extras section. After L37 I would add

```ini
baseline =
    GitPython>=3.1.30
```

Reply:
That makes sense. I was reading the docs which stated the extra in the [] had to be a package. But I assume that extras acts as an alias of sorts.

Reply:
Yes the extra name can be anything. See the old security extra for requests